vllm/entrypoints/llm.py

```diff
@@ -97,9 +97,6 @@ class LLM:
             throughput. However, if the value is too high, it may cause out-of-
             memory (OOM) errors.
         swap_space: The size (GiB) of CPU memory per GPU to use as swap space.
-            This can be used for temporarily storing the states of the requests
-            when their `best_of` sampling parameters are larger than 1. If all
-            requests will have `best_of=1`, you can safely set this to 0.
             Otherwise, too small values may cause out-of-memory (OOM) errors.
```

Comment:
Does this sentence make sense without the deleted paragraph?

Reply:
You're right; no, but I think it would make sense to do it separately in the followup together

Reply:
Ok, we should keep the warning. Just change the structure a little because when I read it I thought "If this is otherwise, what comes before?"

```suggestion
           Too small values may cause out-of-memory (OOM) errors.
```